buildenv/jenkins/variables/defaults.yml

```diff
@@ -170,9 +165,8 @@ boot_jdk_default:
     version:
       8: '8'
       11: '11'
-      15: '15'
       16: '15'
```

Comment:
This can change to 16 as well, then we don't need to have 15 on the build machines any more.

Reply:
I suppose the risk is low that we have (or mask) build problems using the newer bootjdk; I'll update that.

Reply:
Done.